packages/css/src/components/header/README.md

```diff
@@ -2,13 +2,22 @@
 
 # Header
 
-Arranges the City’s logo, the name of the application, and a page menu.
+Displays the City’s logo and a navigation menu at the top of every page.
+Includes the name of the application if it is not the general website.
 
 ## Guidelines
 
-- Use the Header when the site is hosted on amsterdam.nl or one of its subdomains.
-- The page menu can contain a maximum of 5 items, including menu and search.
-- When you have a long subsite title, use no or as few page menu items as possible.
+The Header must be used on all websites and applications for the City of Amsterdam.
+It includes the logo of the City or the organization, the site title (except for the general website), and a menu with links to commonly used pages.
+
+The Header is important because it conveys our corporate identity and is the first thing people see.
+Using it consistently helps users recognize and trust the website.
+It is the same on every page of the application.
+
+The amount of links in the Page Menu should be limited to a handful.
```

Comment:
I think we have differing opinions on this: 'a handful' is way too vague imo, max 5 is a more straightforward guideline I think.